conda_smithy/configure_feedstock.py

```diff
@@ -1938,22 +1883,6 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
                 f"{target_platforms}"
             )
 
-    if (
-        "build_with_mambabuild" in file_config
-        and "conda_build_tool" not in file_config
-    ):
-        warnings.warn(
-            "build_with_mambabuild is deprecated, use conda_build_tool instead",
-            DeprecationWarning,
-        )
-        config["conda_build_tool"] = (
-            "mambabuild" if config["build_with_mambabuild"] else "conda-build"
-        )
-    if file_config.get("conda_build_tool_deps"):
-        raise ValueError(
-            "Cannot set 'conda_build_tool_deps' directly. "
-            "Use 'conda_build_tool' instead."
-        )
     valid_build_tools = (
```

Comment:
It feels like this block (1886-1908) should be covered by the schema, not Python code. Can you double check?